tests/queries/0_stateless/02416_keeper_map.sql

```diff
@@ -0,0 +1,37 @@
+-- Tags: no-ordinary-database, no-fasttest, no-parallel
```

Comment:
this test created 100x more keys (`01686_rocksdb` for reference) but the operations started to time out on ASAN builds. When it was run in parallel, it affected other tests using Keeper, it was probably overloaded.
First I added `no-parallel` but now I use much fewer keys because it was still too heavy so maybe it's okay to run it in parallel. I'll try removing it.